django_scheduled_tasks/scheduler.py

```diff
@@ -0,0 +1,38 @@
+import os
+from apscheduler.schedulers.background import BackgroundScheduler
+from .models import ScheduledTask
+
+_scheduler = BackgroundScheduler()
+
+
+def start_scheduler():
+    """ Start our background scheduler """
```

Comment:
same comment here as before about doc strings starting on new lines for consistency with others that already are

Reply:
:-) https://github.com/bear-rsg/django-scheduled-tasks/pull/1#discussion_r1107278147